rmgpy/rmg/main.py

```diff
@@ -681,6 +682,15 @@ def execute(self, **kwargs):
                 )
                 
                 plotSensitivity(self.outputDirectory, index, reactionSystem.sensitiveSpecies)
+
+        # generate Cantera files chem.cti & chem_annotated.cti in a designated `cantera` output folder
+        try:
+            self.generateCanteraFiles(os.path.join(self.outputDirectory, 'chemkin', 'chem.inp'))
+            self.generateCanteraFiles(os.path.join(self.outputDirectory, 'chemkin', 'chem_annotated.inp'))
+        except EnvironmentError:
+            logging.info('Error: Could not generate Cantera files due to EnvironmentError. Check read\write privileges in output directory.')
+        except:
+            raise
```

Comment:
Sorry, more minor points. You could use `logging.error` instead of writing `logging.info('Error...')`.

Also, there's no need to do `except: raise`, because Python will raise the error regardless. You can just remove those two lines. You generally use try...except blocks if you're going to do something about the error besides just raising it (eg. printing a more helpful error message, or somehow fixing the error).